dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/ConfigActuatorXMLOverrideTests20.java

```diff
@@ -136,8 +150,13 @@ public String getLogMethodName() {
     }
 
     @After
-    public void stopOverrideServer() throws Exception {
-        super.stopServer();
+    public void stopTestServer() throws Exception {
+        String methodName = testName.getMethodName();
+        if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+            super.stopServer(true, "CWWKT0015W");
+        } else {
+            super.stopServer();
```

Comment:
Why not calling `super.stopServer(true)` here.

Reply:
super.stopServer() will call super.stopServer(true)